src/Command/Cache/RebuildCommand.php

```diff
@@ -44,7 +49,7 @@ class RebuildCommand extends Command
      *
      * @param DrupalApi    $drupalApi
      * @param Site         $site
-     * @param $classLoader
+     * @param ClassLoader  $classLoader
```

Comment:
I recall having some issues when type-hinting the 

We had like that but removed on this PR https://github.com/hechoendrupal/drupal-console/pull/2622/files

Will try to find the issue

Reply:
Didn't know about, I will remove it.